packages/rest-explorer/src/rest-explorer.component.ts

```diff
@@ -3,26 +3,28 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {inject} from '@loopback/context';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import {bind, config, ContextTags, inject} from '@loopback/context';
 import {Component, CoreBindings} from '@loopback/core';
-import {RestApplication, createControllerFactoryForClass} from '@loopback/rest';
+import {createControllerFactoryForClass, RestApplication} from '@loopback/rest';
+import {ExplorerController} from './rest-explorer.controller';
 import {RestExplorerBindings} from './rest-explorer.keys';
 import {RestExplorerConfig} from './rest-explorer.types';
-import {ExplorerController} from './rest-explorer.controller';
 
 const swaggerUI = require('swagger-ui-dist');
 
 /**
  * A component providing a self-hosted API Explorer.
  */
+@bind({tags: {[ContextTags.KEY]: RestExplorerBindings.COMPONENT.key}})
 export class RestExplorerComponent implements Component {
   constructor(
     @inject(CoreBindings.APPLICATION_INSTANCE)
     private application: RestApplication,
-    @inject(RestExplorerBindings.CONFIG, {optional: true})
-    config: RestExplorerConfig = {},
+    @config()
```

Comment:
Just double check: is `{optional: true}` also needed for `@config()`? or `@config()` allows empty value?

Reply:
I believe `@config` is setting `optional: true` by default. Could you please confirm, @raymondfeng?